ci/test_python_common.sh

```diff
@@ -20,6 +18,8 @@ FILE_KEY=$1
 rapids-dependency-file-generator \
   --output conda \
   --file-key "${FILE_KEY}" \
+  --prepend-channel "${CPP_CHANNEL}" \
+  --prepend-channel "${PYTHON_CHANNEL}" \
```

Comment:
I think these channels missing was the cause of what we observed in https://github.com/rapidsai/cudf/pull/17994#issuecomment-2657919047, where it looks like the recently-built-in-CI packages were being ignored in favor of older packages from the `rapidsai-nightly` channel.